cmd/statusd/main.go

```diff
@@ -133,6 +134,10 @@ func main() {
 		logger.Error("Node start failed", "error", err)
 		return
 	}
+	
+	// systemd aliveness notifications
+	go startSystemDWatchdog()
+	sdnotify.Ready()
```

Comment:
What if we do not have it as a systemd service? This can run wherever like macOS as well?

Reply:
This has only an effect on Linux:
https://github.com/okzk/sdnotify/blob/master/notify_linux.go
On other systems it does nothing:
https://github.com/okzk/sdnotify/blob/master/notify.go
This is achieved using Go [build constraints](https://golang.org/pkg/go/build/#hdr-Build_Constraints):
```golang
// +build !linux
```